src/service/iot_config.proto

```diff
@@ -263,7 +273,11 @@ message route_stream_req_v1 {
 
 message route_stream_res_v1 {
   action_v1 action = 1;
-  route_v1 route = 2;
+  oneof data {
+    route_v1 route = 2;
+    eui_pair_v1 eui_pair = 3;
+    devaddr_range_v1 devaddr_range = 4;
+  }
```

Comment:
wondering if we need to have the config server signing all of these responses to ensure the client has the option to verify the responses haven't been spoofed by some bad actor?